src/main/java/org/folio/consortia/client/UserTenantsClient.java

```diff
@@ -12,4 +13,7 @@ public interface UserTenantsClient {
 
   @PostMapping(consumes = MediaType.APPLICATION_JSON_VALUE)
   void postUserTenant(@RequestBody UserTenant userTenant);
+
+  @DeleteMapping
+  void deleteUserTenants();
```

Comment:
https://issues.folio.org/browse/MODUSERS-411 this is being implementing in scope this ticket